tests/test_fitsChan.py

```diff
@@ -13,6 +13,58 @@ def pad(card):
     return "%-80s" % (card,)
 
 
+def writeFitsWcs(frameSet, extraOptions=None):
+    """Write a FrameSet as FITS-WCS
+
+    extraOptions are in addition to Encoding=Fits-WCS, CDMatrix=1
+    """
+    options = "Encoding=FITS-WCS, CDMatrix=1"
+    if extraOptions is not None:
+        options = "%s, %s" % (options, extraOptions)
+    fc = ast.FitsChan(ast.StringStream(), options)
+    fc.write(frameSet)
+    return fc
+
+
+def insertPixelMapping(mapping, frameSet):
+    """Make a new WCS by inserting a new mapping at the beginnning of the GRID-IWC mapping
+
+    Return the new FrameSet (the original is not altered).
+    """
+    frameSet = frameSet.copy()
+
+    skyFrame = frameSet.getFrame(ast.FrameSet.CURRENT)  # use this copy for the new sky frame
+    assert isinstance(skyFrame, ast.SkyFrame)
```

Comment:
If this function was a method in the testcase then this could be `self.assertIsInstance()`. Does it need to be a free function?